### PR TITLE
Update padding on settings page cards.

### LINKED ISSUE #1950: Adjust padding on cards
## Description

Adjust the padding on the cards of the settings.

For background info: p1622132280144800-slack-CU6SYV31A

> Since these cards are in the same section, I’d prefer us to use a smaller padding (f.e. 24 px instead of the current 32 px). It’ll come in handy for the advanced settings, too. :)

### PR DESCRIPTION
**Pull request by tommyshellberg**
Fixes #1950 

#### Changes proposed in this Pull Request

The padding applied to <CardBody> elements is added through a property named `size` which accepts things like "medium", "large", etc. When the value is "large", as is the case on the new settings page, it adds the class `is-size-large` and a consequent `padding: 24px 32px` CSS rule to the CardBody. 

https://github.com/Automattic/woocommerce-payments/blob/develop/client/settings/general-settings/index.js#L38

This PR adds some overriding CSS to set the padding to 24px all around based on design feedback.
It would have been nice to remove the usage of the `size` property completely but it has a default of `medium` if not set so we would have to override it anyways to get 24 px padding.

https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/card/body.js#L14

This should only apply to the Cards and CardBody elements on the settings page since it targets cards that are children of the `settings-section` class. 

#### Testing instructions

With this branch checked out and plugin recompiled, you should see a tighter alignment of the content within a settings section which lines up better with the first card on the page.

It's subtle but it's there 😄 

Screen Shot 2021-06-02 at 4 53 44 PM

Screen Shot 2021-06-02 at 4 53 36 PM

I don't see any tests covering this and it doesn't seem to me that a test needs to cover this change. All existing tests pass.
Not adding a changelog since it's just a minor display change.
Looks good on mobile.

-------------------

- [ ] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

**Comment by frosso** [comment]
I think the intent was to use the "medium" size, rather than changing the padding on all directions - could you please confirm with UX whether this is accurate? I'd hate to create some inconsistencies with the Gutenberg components' spacing.

The "Express checkouts" page doesn't seem to be affected by this change: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments_digital_wallets
Should all cards be consistently spaced, then?

**Comment by tommyshellberg** [comment]
I like the idea of just making every card have a size of `medium` with 16px 24px padding - since it's the default value for <CardBody> we can omit it unless there is a reason we want to explicitly pass `medium`.

File: client/settings/test-mode-settings/style.scss [hunk 1]
```diff
@@ -2,4 +2,8 @@
 	> :first-child {
 		margin-top: 0;
 	}
+
+	h4 {
```
Comment (tommyshellberg) [inline client/settings/test-mode-settings/style.scss:6]: I included this to cleanup the extra margin showing at the top of the Test Mode card:

Screen Shot 2021-06-02 at 4 53 44 PM

**Review by frosso — APPROVED** [review]
LGTM

**Pull request merged**